beam/README.md

```diff
@@ -8,12 +8,9 @@ Current interpreter implementation supports the static repl. It compiles the cod
 You have to first build the Beam interpreter by enable the **beam** profile as follows:
 
 ```
-mvn clean package -Pbeam -DskipTests
+mvn clean package -Pbeam -DskipTests -Pscala-2.10
```

Comment:
Hi, you removed the `### Notice` section and added `-Pscala-2.10`. 
It's quite unclear to use scala 2.10 only or not. Does scala-2.10 is not mandatory as of beam 2.0?